sdk/search/azure-search-documents/src/main/java/com/azure/search/annotation/FieldIgnore.java

```diff
@@ -0,0 +1,14 @@
+package com.azure.search.annotation;
```

Comment:
Is this auto-generated? Trying to understand why we would use this annotation without any documentation.

Reply:
I am working with search team to get clear about how customer uses the annotation. I will add doc after major workflow works.